Runtime/Tracking/Follow/Modifier/Property/Rotation/RotateAroundAngularVelocity.cs

```diff
@@ -30,6 +30,13 @@ public class RotateAroundAngularVelocity : PropertyModifier
         [Serialized]
         [field: DocumentedByXml]
         public Vector3State ApplyToAxis { get; set; }
+        
+        /// <summary>
+        /// When true, transforms the angular velocity to be in target's space instead of world space.
+        /// </summary>
+        [Serialized]
+        [field: DocumentedByXml]
+        public bool InTargetSpace { get; set; }
```

Comment:
Nice solution to prevent any regression issues, but I think this should be defaulted to `true` and any regressions can just have this set to `false` so just change this to:

`public bool InTargetSpace { get; set; } = true;`

Reply:
I disagree that this should be "true" by default, because it does notably change the output of the function. If someone is using this script elsewhere, this little change could severely impact them, whereas for the purposes of using the angular transform drive as a twistable object, there will already be additional setup steps, so adding one more of setting this value to true is trivial.

Reply:
yeah ok, i agree with this